### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by Forgind**

### CODE REVIEW

File: src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs [hunk 1]
```diff
@@ -213,9 +213,43 @@ public void UpdateToWorkloadSetVersionWithManifestsNotAvailable()
...
+            InstallSdk();
+
+            var versionToUpdateTo = "8.0.300-preview.0.24217.2";
+            var directory = "C:\\SdkTesting";
+
+            string originalVersion = GetWorkloadVersion();
+
+            var rollback = GetRollback(directory);
+
+            VM.WriteFile("C:\\SdkTesting\\global.json", @"{""sdk"":{""workloadVersion"":""versionToUpdateTo""}}".Replace("versionToUpdateTo", versionToUpdateTo)).Execute().Should().Pass();
```
Comment (marcpopMSFT) [inline src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs:228]: nit: is there a way to use the variable inline rather than the clunky replace logic? Just a bit wordy.

**Review by marcpopMSFT — APPROVED** [review]

File: src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs (hunk 1 above)
Comment (Forgind) [reply src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs:228]: String interpolation is a bit hampered when the string you're creating actually needs { and }. I can try to see if there's an escape sequence for that, though.

File: src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs [hunk 2]
```diff
@@ -213,9 +213,43 @@ public void UpdateToWorkloadSetVersionWithManifestsNotAvailable()
...
+            InstallSdk();
+
+            var versionToUpdateTo = "8.0.300-preview.0.24217.2";
+            var directory = "C:\\SdkTesting";
+
+            string originalVersion = GetWorkloadVersion();
+
+            var rollback = GetRollback(directory);
+
+            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo""}}}").Execute().Should().Pass();
```
Comment (Forgind) [inline src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs:228]: This needed duplication:
```suggestion
            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo""}}}}}").Execute().Should().Pass();
```

File: src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs [hunk 3]
```diff
@@ -213,6 +213,51 @@ public void UpdateToWorkloadSetVersionWithManifestsNotAvailable()
...
+            InstallSdk();
+
+            var versionToUpdateTo = "8.0.300-preview.0.24217.2";
+            var directory = "C:\\SdkTesting";
+
+            string originalVersion = GetWorkloadVersion();
+
+            var rollback = GetRollback(directory);
+
+            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo""}}}}}").Execute().Should().Pass();
```
Comment (Forgind) [inline src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs:228]: Misplaced }
```suggestion
            VM.WriteFile("C:\\SdkTesting\\global.json", @$"{{""sdk"":{{""workloadVersion"":""{versionToUpdateTo}""}}}}").Execute().Should().Pass();
```
